plugins/node/opentelemetry-instrumentation-mongodb/src/types.ts

```diff
@@ -43,6 +56,17 @@ export type CursorState = { cmd: MongoInternalCommand } & Record<
   unknown
 >;
 
+export interface MongoResponseHookInformation {
+  data: CommandResult;
+}
+
+// https://github.com/mongodb/node-mongodb-native/blob/3.6/lib/core/connection/command_result.js
+export type CommandResult = {
+  result?: object;
+  connection?: object;
+  message?: object;
```

Comment:
I'm not a typescript expert, but I think `any` is usually used and not `object`.
Not sure if it's appropriate here though

Reply:
object in this case means all non-primitive types, so if result, connection or message could be a string, number etc then these should be changed to either explicitly include as a possible types or generically use ```unknown``` or ```any```

Reply:
Thanks for this! Changed to `unknown`.